pkg/loop/plugin_median.go

```diff
@@ -17,12 +17,14 @@ import (
 	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
 	pb "github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/pb"
 	"github.com/smartcontractkit/chainlink-relay/pkg/types"
+	"github.com/smartcontractkit/chainlink-relay/pkg/utils"
 )
 
 // PluginMedianName is the name for [PluginMedian]/[NewGRPCPluginMedian].
 const PluginMedianName = "median"
 
 type PluginMedian interface {
+	types.Service
 	NewMedianPluginFactory(ctx context.Context, provider types.MedianProvider, dataSource, juelsPerFeeCoin median.DataSource, errorLog ErrorLog) (libocr.ReportingPluginFactory, error)
```

Comment:
aside -- i believe i'm mildly dyslexic. intermixing names like AB with BA (in this case PluginMedian with NewMedianPlugin) makes it extremely difficult for me to follow.

Reply:
I'll look at these again, but I'm afraid things will unwravel if we switch one or the other - they aren't arbitrary and we don't control all the code that we connect to.